- Only allow system fmtlib usage
- Preserve header-only API with system fmtlib

--- include/spdlog/fmt/fmt.h.orig	2019-01-18 10:13:07 UTC
+++ include/spdlog/fmt/fmt.h
@@ -10,13 +10,13 @@
 // By default spdlog include its own copy.
 //
 
-#if !defined(SPDLOG_FMT_EXTERNAL)
 #ifndef FMT_HEADER_ONLY
 #define FMT_HEADER_ONLY
 #endif
 #ifndef FMT_USE_WINDOWS_H
 #define FMT_USE_WINDOWS_H 0
 #endif
+#if 0 //!defined(SPDLOG_FMT_EXTERNAL)
 #include "bundled/core.h"
 #include "bundled/format.h"
 #else // external fmtlib
--- include/spdlog/fmt/ostr.h.orig	2019-01-18 10:13:07 UTC
+++ include/spdlog/fmt/ostr.h
@@ -7,10 +7,10 @@
 //
 // include bundled or external copy of fmtlib's ostream support
 //
-#if !defined(SPDLOG_FMT_EXTERNAL)
 #ifndef FMT_HEADER_ONLY
 #define FMT_HEADER_ONLY
 #endif
+#if 0 //!defined(SPDLOG_FMT_EXTERNAL)
 #include "bundled/ostream.h"
 #include "fmt.h"
 #else
